Description: sizeof(ws) != wide characters in ws, this will trigger an
 abort when compiled with -D_FORTIFY_SOURCE=2
Author: Kees Cook <kees@ubuntu.com>

Index: bsdmainutils-8.2.1/usr.bin/ncal/ncal.c
===================================================================
--- bsdmainutils-8.2.1.orig/usr.bin/ncal/ncal.c	2011-01-21 19:17:57.663150640 -0800
+++ bsdmainutils-8.2.1/usr.bin/ncal/ncal.c	2011-01-21 19:18:15.113588903 -0800
@@ -692,7 +692,7 @@
 				wprintf(L"%-*ls  ",
 				    mw, wcenter(ws, year[i].name, mw));
 			else {
-				swprintf(ws, sizeof(ws), L"%-ls %d",
+				swprintf(ws, MAX_WIDTH, L"%-ls %d",
 				    year[i].name, M2Y(m + i));
 				wprintf(L"%-*ls  ", mw, wcenter(ws1, ws, mw));
 			}
